eman2/protocols/protocol_tomo_extraction.py

```diff
@@ -160,6 +161,16 @@ def extractParticles(self):
                      self._getExtraPath(pwutils.replaceBaseExt(tomo, 'hdf')))
             cleanPath(self._getExtraPath("sptboxer_01"))
 
+    def convertOutput(self):
+        program = eman2.Plugin.getProgram('e2proc3d.py')
+        for hdfFile in glob.glob(self._getExtraPath('*.hdf')):
+            args = ' --unstacking'
+            args += ' %s' % abspath(hdfFile)
+            args += ' %s' % abspath(self._getExtraPath(pwutils.replaceBaseExt(hdfFile, 'mrc')))
+            self.runJob(program, args, cwd=self._getExtraPath(),
+                        numberOfMpi=1, numberOfThreads=1)
+            os.remove(hdfFile)
```

Comment:
I would advocate using our libraries: https://scipion-em.github.io/docs/docs/developer/python-style-guide#use-library-functions. one of pwutils funcs should work